### PR TITLE
Create community page updates, Cosmos/DAO support.

### PR DESCRIPTION
**Pull request by jnaviask**
## Description

This PR builds on top of James' work in moving the create community modal into a page. It does the following:
- Moves create community modal into a page, with separate files for each tab (chain).
- Refactors the "standard" rows (e.g. website, etc) into a separate utility file.
- Adds Cosmos and ETH DAO support (refactoring ETH-specific rows into the utility file as well), both gated on admin.
- Adds ETH chain ID dropdown using chainid.network/chains.json to populate readable names.
- Gates ERC20 creation with custom URL on admin (see: routes/createChain.ts).

## Motivation and Context

Provides far better self-serve functionality to BD team for onboarding, and cleans up user ERC20 onboarding as well.

## How has this been tested?

Attempted validation and creation with a bunch of different chains/networks and confirmed working. Needs more QA esp on routes.

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [x] yes, but I did not run tests
- [ ] no

### CODE REVIEW

File: client/scripts/app.ts [hunk 1]
```diff
@@ -397,11 +397,12 @@ export function initCommunity(communityId: string): Promise<boolean> {
 }
 
 export async function initNewTokenChain(address: string) {
-  const response = await $.getJSON('/api/getTokenForum', { address });
+  const response = await $.getJSON('/api/getTokenForum', { address, autocreate: true });
   if (response.status !== 'Success') {
     // TODO: better custom 404
     m.route.set('/404');
   }
+  console.log(response.result);
```
Comment (zakhap) [inline client/scripts/app.ts:405]: log

**Comment by zakhap** [comment]
While i'm still mid review, first thought:
- we should deprecate the Offchain Community Creation tab here.

File: client/scripts/views/components/sidebar/index.ts [hunk 2]
```diff
@@ -622,7 +626,7 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
             window.open('https://commonwealth.im/');
           },
         }),
-      ]),
+      ]): '',
```
Comment (zakhap) [inline client/scripts/views/components/sidebar/index.ts:629]: is this better than `null`?

File: client/scripts/views/pages/create_community/index.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    // query eth chains
+    $.get(`${app.serverUrl()}/getSupportedEthChains`, {}).then(async (res) => {
+      if (res.status === 'Success') {
+        vnode.state.ethChains = res.result;
+      }
+
+      // query names from chainlist if possible
+      const chains = await $.getJSON('https://chainid.network/chains.json');
+      console.log(chains);
```
Comment (zakhap) [inline client/scripts/views/pages/create_community/index.ts:56]: log

File: client/scripts/views/pages/create_community/erc20_form.ts [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+      if (!vnode.state.address || !vnode.state.chain_id) return;
+      vnode.state.loading = true;
+      const args = {
+        address: vnode.state.address,
+        chain_id: vnode.state.chain_id,
+        allowUncached: true,
+      };
+      try {
+        const res = await $.get(`${app.serverUrl()}/getTokenForum`, args);
+        console.log(res);
```
Comment (zakhap) [inline client/scripts/views/pages/create_community/erc20_form.ts:60]: log

**Review by zakhap — COMMENTED** [review]
Can't seem to get these three views to show up in the tabs:
- Cosmos
- eth daos
- substrate
I'm aware they're supposed to be site-admin features, but they don't seem to render locally when that's true either

File: client/scripts/views/pages/create_community/eth_dao_form.ts [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+        if (vnode.state.network === ChainNetwork.Compound) {
+          const provider = new Web3.providers.WebsocketProvider(vnode.state.url);
+          const compoundApi = new CompoundAPI(
+            null,
+            vnode.state.address,
+            provider,
+          );
+          await compoundApi.init(vnode.state.token_name);
+          if (!compoundApi.Token) {
+            throw new Error('Could not find governance token. Is "Token Name" field valid?');
```
Comment (zakhap) [inline client/scripts/views/pages/create_community/eth_dao_form.ts:72]: we should also suggest to the user "Try all lowercase if you're sure on spelling" or something similar. I was testing with COMP and it needed to be 'comp'

Comment (jnaviask) [reply client/scripts/views/pages/create_community/eth_dao_form.ts:72]: I will add a "case sensitive" note.

**Review by zakhap — APPROVED** [review]
Worked for me on: 
- Aave/Comp
- Cosmo
- ERC20
- Substrate

**Pull request merged**